apps/meteor/app/api/server/v1/subscriptions.ts

```diff
@@ -103,3 +107,35 @@ API.v1.addRoute(
 		},
 	},
 );
+
+API.v1.addRoute(
+	'subscriptions.exists',
```

Comment:
I was thinking of moving this to a new `rooms.isMember`, this would make more sense. WDYT?

Reply:
Yes this makes sense.